include/villas/nodes/iec61850_goose.hpp

```diff
@@ -0,0 +1,184 @@
+/** Node type: IEC 61850 - GOOSE
+ *
+ * @author Philipp Jungkamp <philipp.jungkamp@rwth-aachen.de>
+ * @copyright 2023, Institute for Automation of Complex Power Systems, EONERC
+ * @license Apache 2.0
+ *********************************************************************************/
+
+#pragma once
+
+#include <string>
+#include <optional>
+#include <cstdint>
+#include <ctime>
+#include <array>
+#include <map>
+#include <villas/node/config.hpp>
+#include <villas/node.hpp>
+#include <villas/pool.hpp>
+#include <villas/queue_signalled.h>
+#include <villas/signal.hpp>
+#include <libiec61850/goose_receiver.h>
+#include <libiec61850/goose_subscriber.h>
+
+namespace villas {
+namespace node {
+namespace iec61850 {
+
+// A GooseValue is a SignalData value with attached Metadata for the MmsType and SignalType
+class GooseValue {
```

Comment:
I would propose to name this `GooseSignal` as this is more aligned with the `Signal` type in VILLASnode.

Or do you have a specific reason for naming it value?

Reply:
I originally had the GooseValue wrap a MmsValue instead of SignalData. I already thought about renaming to MmsSignal. Though GooseSignal makes more sense for now.